test/Microsoft.AspNetCore.DataProtection.Extensions.Test/DataProtectionProviderTests.cs

```diff
@@ -38,6 +38,46 @@ public void System_UsesProvidedDirectory()
         [ConditionalFact]
         [ConditionalRunTestOnlyIfLocalAppDataAvailable]
         [ConditionalRunTestOnlyOnWindows]
+        public void System_UsesDefaultDirectory()
+        {
+            var keysPath = Path.Combine(Environment.ExpandEnvironmentVariables("%LOCALAPPDATA%"), "ASP.NET", "DataProtection-Keys");
```

Comment:
This is pretty evil, but until the code is better factored we kinda need it. Do good tests for the other overloads?